elliott/elliottlib/bzutil.py

```diff
@@ -421,6 +422,7 @@ def _get_severity(self):
         return None
 
     def all_advisory_ids(self):
+        errata.sync_jira_issue(self.id)
```

Comment:
Think `all_advisory_ids` gets called pretty often, and it is almost never needed to ask errata to update its list. 

If a jira is attached to an advisory, errata will know. 

If the next version does not ship next week, we will soon not check: 

Remains the case where we prepare 4.y and 4.(y+1), and we prepare 4.y earlier than 4.(y+1). 

Suggesting the optimization to first get `all_advisory_ids`, then if empty or error, sync and try again.

Reply:
updated in https://github.com/openshift-eng/art-tools/pull/1063/commits/2e297444788301fb704f7cf0fcad88cceccfaef0